src/algebra/group_power/lemmas.lean

```diff
@@ -105,6 +105,36 @@ lemma zsmul_one [has_one A] (n : ℤ) : n • (1 : A) = n := by cases n; simp
 
 end sub_neg_monoid
 
+section division_monoid
+variables [division_monoid α]
+
+-- note that `mul_zsmul` and `zpow_mul` have the primes swapped since their argument order
+-- and therefore the more "natural" choice of lemma is reversed.
```

Comment:
(Did I parse the sentence correctly?)
```suggestion
-- Note that `mul_zsmul` and `zpow_mul` have the primes swapped since their argument order,
-- and therefore the more "natural" choice of lemma, is reversed.
```

Reply:
Yup, although youn should only half trust me because I didn't touch this comment.